core/metrics-core-impl/src/main/java/org/hawkular/metrics/core/impl/cassandra/MetricsServiceCassandra.java

```diff
@@ -800,6 +801,40 @@ private void dropKeyspace(String keyspace) {
         });
     }
 
+    @Override
```

Comment:
I will open a separate PR for the change to SimpleDataCallback and create the JIRA ticket.